ch.tigerjython.TigerJython2.yml

```
@@ -0,0 +1,49 @@
+id: ch.tigerjython.TigerJython2
+branch: '2.39'
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk/install.sh
+  - name: tigerjython2
+    buildsystem: simple
+    build-commands:
+      - install -Ddm755 /app/tigerjython/Lib
+      - install -Ddm755 /app/tigerjython/TestSamples
+      - sed -i "s|/usr/bin/java|/app/jre/bin/java|g" tigerjython
+      - sed -i "s|/opt/tigerjython|/app/tigerjython|g" tigerjython
+      - install -Dm644 Lib/* /app/tigerjython/Lib
+      - install -Dm644 TestSamples/* /app/tigerjython/TestSamples
+      - install -m755 tigerjython2.jar /app/tigerjython/tigerjython2.jar
+      - install -m755 tigerjython /app/tigerjython/tigerjython
+
+    sources:
+      - type: archive
+        url: https://git.jdmweb2.ch/beat/tigerjython_pkg/raw/tag/V2.39/TigerJython.tar.gz
+        sha256: 5961b11112ad27d4831d47d3422baa41d1ba8059e8dde3db2e4c4a8759aad607
+finish-args:
+  - --env=PATH=/app/tigerjython:/app/jre/bin:/usr/bin
+  # X11 + XShm access
+  - --share=ipc
+  # - --socket=fallback-x11
+  # Wayland access
+  - --socket=x11
+  # GPU acceleration if needed
+  - --device=dri
+  # Needs to talk to the network:
+  - --share=network
+  # Needs to save files locally
+  - --filesystem=xdg-documents
+
+## Error: Command 'tigerjython' not found
+# command: tigerjython
+
+## WARNING: toplevel-command-is-path
+## https://github.com/flathub-infra/flatpak-builder-lint/issues/23
+command: /app/tigerjython/tigerjython
```

```suggestion
command: tigerjython
```

should work with the symlink to /app/bin

---

`command` is found above `finish-args` which is found above `modules`

---

The error is because we expect the command to be in the path (`/app/bin`) which is the standard way.